app/assets/scss/_service-status.scss

```diff
@@ -0,0 +1,23 @@
+.service-status,
+%service-status {
+  font-weight: bold;
+}
+
+.service-status-removed,
+.service-status-disabled {
+  @extend %service-status;
+  color: $red;
+}
+
+.service-status-published,
+.service-status-public {
+  @extend %service-status;
+  color: $button-colour;
+}
+
+.service-status-unpublished,
```

Comment:
This triplet of indecisiveness doesn't look great to me, but I'll hopefully never look at a `.scss` file so probably don't care that much.